srcpkgs/ruby-xdg/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'ruby-xdg'
 pkgname=ruby-xdg
-version=5.1.1
-revision=2
+version=6.5.0
+revision=1
 build_style=gem
 short_desc="Module for supporting the XDG Base Directory Standard"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="BSD-2-Clause"
-homepage="http://rubyworks.github.com/xdg"
-checksum=3d050ea585c0d1fdb69bbb9ee58858edbd7ad6e6db830391e08b174eddbe4b15
+license="Hippocratic-2.1"
+homepage="http://rubyworks.github.io/xdg"
+changelog="https://raw.githubusercontent.com/rubyworks/xdg/master/HISTORY.md"
```

Comment:
seems I was looking at the wrong upstream earlier
```suggestion
homepage="https://www.alchemists.io/projects/xdg/"
changelog="https://raw.githubusercontent.com/bkuhlmann/xdg/main/README.adoc"
```

Reply:
I've changed again to your suggestion. :+1: